appengine/README.md

```diff
@@ -0,0 +1,101 @@
+# Google App Engine Standard Environment Java Samples
+
```

Comment:
You can put a table of contents at the top to help guide the process.  see github.com/googlecloudplatform/cloud-bigtable-examples/